doc/api/fs.md

```diff
@@ -1280,8 +1280,8 @@ changes:
 -->
 
 * `file` {string|Buffer|URL|FileHandle} filename or `FileHandle`
-* `data` {string|Buffer|TypedArray|DataView|Object|AsyncIterable|Iterable
-  |Stream}
+* `data` {string|Buffer|TypedArray|DataView|AsyncIterable|Iterable|Stream|
+Object}*
```

Comment:
You need to keep this on one line, otherwise the parser doesn't recognize this as a list of types
```suggestion
* `data` {string|Buffer|TypedArray|DataView|AsyncIterable|Iterable|Stream|Object}*
```

Reply:
It was already not 1 line; I merely swapped what was on the second line.

Your suggestion violates the 80 character line length limit :(

Reply:
My bad, this should work:
```suggestion
* `data` {string|Buffer|TypedArray|DataView|AsyncIterable|Iterable|Stream
  |Object} *
```